FrameworkSystem/Client/ComponentMonitoringClient.py

```diff
@@ -0,0 +1,25 @@
+"""
+Class for making requests to a ComponentMonitoring Service
+"""
+
+__RCSID__ = "$Id$"
+
+from DIRAC.Core.Base.Client import Client
+
+class ComponentMonitoringClient( Client ):
+
+  def __init__( self, host, port = None, **kwargs ):
+    """
+    Constructor function. Takes a mandatory host parameter
+    """
+
+    Client.__init__( self, **kwargs )
+    if not port:
+      port = 3426
```

Comment:
I don't think it's a good idea to have a port number in the client (and in general port numbers are not in the python code). The ConfigTemplate.cfg it's the only place where it's stored.